crewai_tools/tools/firecrawl_scrape_website_tool/firecrawl_scrape_website_tool.py

```diff
@@ -53,7 +54,9 @@ class FirecrawlScrapeWebsiteTool(BaseTool):
 
     _firecrawl: Optional["FirecrawlApp"] = PrivateAttr(None)
 
-    def __init__(self, api_key: Optional[str] = None, **kwargs):
+    def __init__(self, api_key: Optional[str] = None, config: Optional[Dict[str, Any]] = None, **kwargs):
+        if config:
+            kwargs["config"] = config
```

Comment:
I got a bit confused here since

```python
config: Dict[str, Any] = Field(
```
means that config is required

however, your initializer is treating is as optional:
```python
config: Optional[Dict[str, Any]] = None
```

Does this change to `config` really needed?